service/src/main/java/bio/terra/catalog/datarepo/DatarepoService.java

```diff
@@ -54,6 +56,19 @@ public Map<String, List<String>> getSnapshotIdsAndRoles(AuthenticatedUserRequest
     }
   }
 
+  public SnapshotModel getPreviewTables(AuthenticatedUserRequest user, String snapshotId) {
+    try {
+      UUID id = UUID.fromString(snapshotId);
+      return snapshotsApi(user)
+          .retrieveSnapshot(
+              id,
+              List.of(
+                  SnapshotRetrieveIncludeModel.DATA_PROJECT, SnapshotRetrieveIncludeModel.TABLES));
```

Comment:
Is `SnapshotRetrieveIncludeModel.DATA_PROJECT` needed? I think we only use the `TABLES` content in the response.

Reply:
Tables is the only thing that was needed, but for some reason `DATA_PROJECT` was also included in the UI. I am going to pull it.